app/controllers/api/v1/merchants_controller.rb

```diff
@@ -20,6 +20,8 @@ def create
     else
       render json: { errors: merchant.errors.messages }, status: 422
     end
+  rescue ActionController::ParameterMissing => error
+    render json: { message: "Creation failed", errors: [error.message] }, status: 422
   end
```

Comment:
This corrects a sad path error when form is empty in POST create a new merchant.